vendor/bat-native-ledger/src/bat_get_media.cc

```diff
@@ -28,12 +28,17 @@ BatGetMedia::~BatGetMedia() {}
 std::string BatGetMedia::GetLinkType(const std::string& url,
                                      const std::string& first_party_url,
                                      const std::string& referrer) {
-  std::string type = "";
+  std::string type = std::string();
```

Comment:
`std:string type;` is all that is required here to initialize a blank string

Reply:
I was just getting rid of empty `""` but I can do unassign init

Reply:
Fixed